articles/resources/relnotes.md

```diff
@@ -17,6 +17,21 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: Sept 29th, 2020*
+
+This release contains the following:
+
+- Performance improvements for simulator: better application of gate fusion decisions; improved parallelization on Linux system; added intelligent scheduling of gate execution; bug fixes; overall speed improvement of ~5x (even faster on larger problems).
+- IntelliSense support is enabled when using Python as a driver.
+- Improved IQ# Python interoperability.
+- Improvements to the Microsoft.Quantum.Arrays namespace (microsoft/QuantumLibraries#313)
```

Comment:
@cgranade @msoeken @guenp could one of you please review/update this release note for the `Microsoft.Quantum.Arrays` improvements?